src/class-metadata.php

```diff
@@ -37,14 +37,15 @@ public function __construct( Parsely $parsely ) {
 	}
 
 	/**
-	 * Creates parsely metadata object from post metadata.
+	 * Creates Parse.ly metadata object from post metadata.
 	 *
-	 * @param array<string, mixed> $parsely_options parsely_options array.
-	 * @param WP_Post              $post object.
+	 * @param WP_Post $post object.
 	 *
 	 * @return array<string, mixed>
 	 */
-	public function construct_metadata( array $parsely_options, WP_Post $post ): array {
+	public function construct_metadata( WP_Post $post ): array {
```

Comment:
This is a breaking change - it's changing the signature of a public method. Why is it in a 3.3.0 release? (unless this method was added after the 3.2 release?)

If you left the tests alone, would this still pass? I suspect it wouldn't. This is a good indicator that custom code in user's code would no longer work correctly.

Reply:
Yes, this method has been added on 3.3.0. Notice that we keep the public method for backwards compatibility: https://github.com/Parsely/wp-parsely/blob/5acb745bdb89d45205bdeb764653d49d9ab75631/src/class-parsely.php#L239

There is an issue to remove this method in 4.0.0